src/status_im2/contexts/quo_preview/profile/profile_card/style.cljs

```diff
@@ -0,0 +1,31 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def card-container
+  {:flex-direction   :column
+   :width            335
```

Comment:
Can we use margin, instead of hard coding width.
[Designs](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=2925%3A328154&t=iIEDCNlTrTZpCp8b-0) are still in WIP, but it seems we can use margin as 20.

Reply:
Sure! I will use the margin in the parent tho since I think the width of the component should always be the full width.

Reply:
Better to add `:margin-horizontal 20` to card-container style itself

Reply:
Mhm I don't agree with that since the screen that this component is going to be rendered, will have a margin-horizontal 20, so the card-container will fit that properly. If I add that margin to this component, and the parent also has the margin, it's not going to be rendered as it should

Reply:
<img width="365" alt="image" src="https://user-images.githubusercontent.com/44204622/210114171-ec7f6cb6-c37e-4496-9f7d-349ff5dfabb6.png">